src/test/java/seedu/address/logic/command/ShowCommandTest.java

```diff
@@ -18,12 +45,112 @@ public void execute_show_success() {
     }
 
     @Test
-    public void execute_goto_failure() {
+    public void execute_showAccommodation_success() {
+        try {
```

Comment:
I wonder why you didn't use the method in CommandTestUtil (assertCommandSuccess(), etc.)?